src/txn/txn_rollback_to_stable.c

```diff
@@ -803,7 +1095,9 @@ __rollback_abort_col_var(WT_SESSION_IMPL *session, WT_REF *ref, wt_timestamp_t r
             kcell = WT_COL_PTR(page, cip);
             __wt_cell_unpack_kv(session, page->dsk, kcell, &unpack);
             rle = __wt_cell_rle(&unpack);
-            __rollback_abort_col_ondisk_kv(session, ref, cip, rollback_timestamp, recno);
+            for (j = 0; j < rle; j++) {
+                __rollback_abort_col_ondisk_kv(session, ref, cip, rollback_timestamp, recno + j);
```

Comment:
This function returns an int and we should handle that.

Reply:
Fixed. I also changed __rollback_abort_col_var to handle the return int as well.